dfx/index.ts

```diff
@@ -92,3 +127,48 @@ export async function getIdentity(
     );
     return Secp256k1KeyIdentity.fromPem(await readFile(identityPath, 'utf-8'));
 }
+
+export function getPemKey(identityName: string = getIdentityName()): string {
```

Comment:
What does getIdentityName do? Is it grabbing the name of the current dfx identity? If so I think we should change its name to indicate that

Reply:
okay